### PR TITLE
NOTICK - Adding keys and timestamp for genesis

### PR DESCRIPTION
**Pull request by sacherjj**
Added in keys for validators from sheet.
Updated chainspec.toml values from updates to production/chainspec.toml.
Set timestamp to 2020-09-18 18:00 UTC (2PM EST, 11AM PST) today.

### CODE REVIEW

File: resources/charlie/chainspec.toml [hunk 1]
```diff
@@ -2,10 +2,10 @@
 # Human readable name for convenience; the genesis_hash is the true identifier.  The name influences the genesis hash by
 # contributing to the seeding of the pseudo-random number generator used in contract-runtime for computing genesis
 # post-state hash.
-name = 'casper-charlienet'
+name = 'casper-charlie-testnet'
 # Timestamp for the genesis block, also used in seeding the pseudo-random number generator used in contract-runtime for
 # computing genesis post-state hash.
-timestamp = 1600218437000
+timestamp = 1600452000000
```
Comment (afck) [inline resources/charlie/chainspec.toml:8]: That's 6 PM GMT, right? (I thought I read 8 PM somewhere.)

File: resources/charlie/chainspec.toml [hunk 2]
```diff
@@ -30,25 +30,25 @@ accounts_path = '/etc/casper/accounts.csv'
 # make key blocks for the upcoming eras.  If the era is over by the time we start the nodes, they'll not be able to
 # produce blocks in it, and there won't be a new era build either.  That means when a completely new network is started,
 # the genesis era start time has to be adjusted to be active at the time.
-genesis_era_start_timestamp = 1600218437000
-# Era duration defined as a fixed number of milliseconds.  604800000 ms = 1 week.
-era_duration_millis = 604800000
+genesis_era_start_timestamp = 0
```
Comment (afck) [inline resources/charlie/chainspec.toml:33]: That must be the same as `timestamp`. (Duplicated; will be removed soon.)

File: resources/charlie/chainspec.toml (hunk 1 above)
Comment (sacherjj) [reply resources/charlie/chainspec.toml:8]: Yes, I was updating the old description from when I thought we might start at 20:00 yesterday.

File: resources/charlie/chainspec.toml (hunk 2 above)
Comment (sacherjj) [reply resources/charlie/chainspec.toml:33]: Oh, good catch.  Messed this up syncing other close value.

**Review by afck — APPROVED** [review]

File: resources/charlie/chainspec.toml [hunk 3]
```diff
@@ -2,10 +2,10 @@
 # Human readable name for convenience; the genesis_hash is the true identifier.  The name influences the genesis hash by
 # contributing to the seeding of the pseudo-random number generator used in contract-runtime for computing genesis
 # post-state hash.
-name = 'casper-charlienet'
+name = 'casper-charlie-testnet'
```
Comment (MParlikar) [inline resources/charlie/chainspec.toml:5]: This is fine for now.  Next time we should include charlie-testnet-1

**Review by MParlikar — APPROVED** [review]

File: resources/charlie/chainspec.toml [hunk 4]
```diff
@@ -30,25 +30,25 @@ accounts_path = '/etc/casper/accounts.csv'
...
 # the way.  10800000 ms = 3 hours.
-entropy_duration_millis = 10800000
+entropy_duration_millis = 0
 # Keep voting on the switch block for a fixed amount of time; effective if the summit level is zero.
 # 172800000 ms = 2 days.
-voting_period_duration_millis = 172800000
+voting_period_duration_millis = 0
 # Integer between 0 and 99 representing the fault tolerance threshold as a percentage, used by the internal finalizer.
-finality_threshold_percent = 10
+finality_threshold_percent = 50
```
Comment (sacherjj) [inline resources/charlie/chainspec.toml:48]:
```suggestion
finality_threshold_percent = 10
```

**Review by afck — APPROVED** [review]

File: resources/charlie/chainspec.toml [hunk 5]
```diff
@@ -43,9 +43,12 @@ booking_duration_millis = 0
 entropy_duration_millis = 0
 # Keep voting on the switch block for a fixed amount of time; effective if the summit level is zero.
 # 172800000 ms = 2 days.
-voting_period_duration_millis = 0
-# Integer between 0 and 99 representing the fault tolerance threshold as a percentage, used by the internal finalizer.
-finality_threshold_percent = 20
+voting_period_duration_millis = 172800000
```
Comment (afck) [inline resources/charlie/chainspec.toml:46]: I think this is currently ignored ([NDRS-364](https://casperlabs.atlassian.net/browse/NDRS-364)), and determined by the `AUCTION_DELAY` contant.

Comment (sacherjj) [reply resources/charlie/chainspec.toml:46]: Just updated to match with resolve conflict with master.

**Pull request merged**